include/grl/flatbuffer/KUKAiiwa.fbs

```diff
@@ -209,6 +210,10 @@ table KUKAiiwaArmConfiguration {
   /// position, cartesian impedence, or joint impedence low level controller adjustments
   controlMode:EControlMode;
 
+  setCartImpedance:CartesianImpedenceControlMode;
```

Comment:
is this not handled by controlMode:EControlMode in the line above?

From KUKAiiwa.fbs:

enum EControlMode : byte {
  POSITION_CONTROL_MODE=0,
  CART_IMP_CONTROL_MODE=1,
  JOINT_IMP_CONTROL_MODE=2,
  NO_CONTROL=3
}